docs/quick-start.md

```diff
@@ -50,7 +50,11 @@ Open a port-forward so you can access the UI:
 kubectl -n argo port-forward deployment/argo-server 2746:2746
 ```
 
-This will serve the UI on <https://localhost:2746>. Due to the self-signed certificate, you will receive a TLS error which you will need to manually approve.
+This will serve the UI on <https://localhost:2746>. 
```

Comment:
@rdgoite Can you please remove the blank space at the end of this line, this is causing the test to fail.

```suggestion
This will serve the UI on <https://localhost:2746>.
```